### PR TITLE
Use FileInput from cozy-ui

### PR DESCRIPTION
**Pull request by cedricmessiant**
See https://github.com/cozy/cozy-ui/pull/1021

Except the aria-label change, I think it's OK. I tried to fix the fact that we display the icon via MenuItem in photos but styles are not easy to fix. Also, I removed translate() in UploadButton in photos because it seems we don't need it.

### CODE REVIEW

File: src/drive/web/modules/upload/UploadButton.jsx [hunk 1]
```diff
@@ -2,51 +2,25 @@ import React from 'react'
...
-
 const UploadButton = ({ label, disabled, onUpload, className }) => (
-  <label
-    role="button"
-    disabled={disabled}
+  <FileInput
     className={className}
-    style={styles.parent}
+    data-test-id="upload-btn"
+    label={label}
```
Comment (y-lohse) [inline src/drive/web/modules/upload/UploadButton.jsx]: Seeing this a bit late, but I guess a `label` prop makes it sound like a label will be visually rendered when it's about a11y. Maybe `aria-label` would have been better?

Comment (cedricmessiant) [reply src/drive/web/modules/upload/UploadButton.jsx]: I agree. I will make a PR in cozy-ui

**Comment by cedricmessiant** [comment]
Except the `aria-label` change, I think it's OK. I tried to fix the fact that we display the icon via `MenuItem` in photos but styles are not easy to fix. Also, I removed `translate()` in `UploadButton` in photos because it seems we don't need it.

**Comment by cedricmessiant** [comment]
[(𝐒) Composant select file](https://trello.com/c/02UskpST/2008-%F0%9D%90%92-composant-select-file)

File: src/drive/web/modules/upload/UploadButton.jsx (hunk 1 above)
Comment (cedricmessiant) [reply src/drive/web/modules/upload/UploadButton.jsx]: well in fact, there is no `label` prop in `FileInput`, thanks :-)

Comment (cedricmessiant) [reply src/drive/web/modules/upload/UploadButton.jsx]: There is no `label` prop in `FileInput` component in fact. So the label we pass here is just passed to the hidden input so I don't think we need this in the end.

**Pull request merged**